docs/source/client_gui/gobject_signals.rst

```diff
@@ -16,44 +16,132 @@ events.
    This signal is emitted when campaign attributes are changed. Subscribers to
    this signal can use it to update and refresh information for the modified
    campaign.
+   Signal emission stage set to run first.
 
+   :Emission Stage: ``RUN_FIRST``
    :param str campaign_id: The ID of the campaign whose information was changed.
 
+.. py:function:: campaign-created(campaign_id)
+
+   This signal is emitted after the user creates a new campaign id. Subscribers
+   to this signal can use it to conduct an action after a new campaign id is created.
+
+   :Emission Stage: ``RUN_FIRST``
+   :param str campaign_id: The ID of the new campaign.
+
+.. py:function:: campaign-deleted(campaign_id)
+
+   This signal is emitted when the user deletes a campaign. Subscribers
+   to this signal can use it to conduct an action after the campaign is deleted.
+
+   :Emission Stage: ``RUN_LAST``
+   :param str campaign_id: The ID of the campaign.
+
 .. py:function:: campaign-set(campaign_id)
 
    This signal is emitted when the user sets the current campaign. Subscribers
    to this signal can use it to update and refresh information for the current
    campaign.
 
-   :param str campaign_id: The ID of the new campaign.
+   :Emission Stage: ``RUN_FIRST``
+   :param str campaign_id: The ID of the campaign.
+
+.. py:function:: config-load(load_defaults)
+
+   This signal is emitted when the client configuration is loaded from disk. This
+   loads all of the clients settings used within the GUI.
+   Signal emission stage set to run last.
+
+   :Emission Stage: ``RUN_FIRST``
+   :param bool load_defaults: Load defaults true or false.
```

Comment:
true or false is implied by the fact that it's a boolean, so this should be more descriptive. Can simple copy the docs from `do_config_load` of "Load missing options from the template configuration file."